lntest/itest/lnd_test.go

```diff
@@ -5171,6 +5171,137 @@ func testListChannels(net *lntest.NetworkHarness, t *harnessTest) {
 
 }
 
+// testSetChannelStatus checks that calls to the SetChannelStatus RPC update
+// the channel graph as expected.
+func testSetChannelStatus(net *lntest.NetworkHarness, t *harnessTest) {
+	ctxb := context.Background()
+
+	// Create two fresh nodes and open a channel between them.
+	alice, err := net.NewNode("Alice", nil)
```

Comment:
You'll need to set some custom arguments here to make `lnd` use lower values for it's gossip/enable/disable timeout intervals. Here's an example from a related test: https://github.com/lightningnetwork/lnd/blob/master/lntest/itest/lnd_test.go#L13112